GroupDocs.Editor.MVC/Products/Editor/Config/EditorConfiguration.cs

```diff
@@ -37,7 +38,9 @@ public EditorConfiguration()
             filesDirectory = valuesGetter.GetStringPropertyValue("filesDirectory", filesDirectory);
             if (!IsFullPath(filesDirectory))
             {
-                filesDirectory = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, filesDirectory);
+                // TODO: check for more cross-platform solution
+                var appRoot = AppContext.BaseDirectory.Substring(0, AppContext.BaseDirectory.LastIndexOf("bin"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5172828)